server/src/main/java/org/apache/iotdb/db/engine/cache/DeviceMetaDataCache.java

```diff
@@ -153,6 +147,29 @@ public static DeviceMetaDataCache getInstance() {
     }
   }
 
+  private void printCacheLog(boolean isHit) {
+    if (!logger.isDebugEnabled()) {
+      return;
+    }
+    if (isHit) {
+      logger.debug(
+          "[ChunkMetaData cache hit] The number of requests for cache is {}, hit rate is {}.",
+          cacheRequestNum.get(), cacheHitNum.get() * 1.0 / cacheRequestNum.get());
+    } else {
+      logger.debug(
+          "[ChunkMetaData cache didn't hit] The number of requests for cache is {}, hit rate is {}.",
+          cacheRequestNum.get(), cacheHitNum.get() * 1.0 / cacheRequestNum.get());
+    }
+  }
```

Comment:
These codes could be more concise.
```suggestion
      logger.debug(
          "[ChunkMetaData cache {}hit] The number of requests for cache is {}, hit rate is {}.",
          isHit ? "":"didn't ", cacheRequestNum.get(), cacheHitNum.get() * 1.0 / cacheRequestNum.get());
```